skimage/measure/_blur_effect.py

```diff
@@ -16,6 +16,9 @@
 __all__ = ['blur_effect']
 
 
+_EPSILON = np.spacing(1)
```

Comment:
Huh, didn't know `spacing` worked this way.

```suggestion
_EPSILON = np.finfo(np.float64).eps
```
feels easier to read and more explicit. But feel welcome to ignore. :wink:

Reply:
Though, if you want to continue using `spacing` it should probably wrap 1 like suggested in the docstring
```suggestion
_EPSILON = np.spacing(np.float64(1))
```

Reply:
I'd also move it into the function itself, since it's not used elsewhere...

Reply:
> I'd also move it into the function itself, since it's not used elsewhere...

I think it's still valuable to have the variable in its own line... It's more readable and, who knows, it may be subject to changes again, especially in the light of these different alternatives (`p.spacing(1)`, `np.spacing(np.float64(1))`, `np.finfo(np.float64).eps`).